app/schemas/com.uxstate.launchpad.data.local.database.LaunchDatabase/1.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
Interesting, how do you create this JSON File?

Reply:
Room exports your database's schema information into a JSON file at compile time, when you set the `room.schemaLocation` annotation processor argument. This allows Room to create older versions of the database and enables automatic migration.

[For more information you can always refer to the official docs.](https://developer.android.com/training/data-storage/room/migrating-db-versions#export-schemas)

Reply:
Awesome, thanks for the tip